src/PlayButton.test.js

```diff
@@ -0,0 +1,89 @@
+// @flow
+
+import React from 'react';
+import Adapter from 'enzyme-adapter-react-16';
+import { configure, shallow } from 'enzyme';
+import { createIntl } from 'react-intl';
+import messages from './messages.json';
+import AriaDisablingButton from './AriaDisablingButton';
+import PlayButton from './PlayButton';
+
+configure({ adapter: new Adapter()});
+
+const defaultPlayButtonProps = {
+    interpreterIsRunning: false,
+    runButtonDisabled: false
+};
+
+function createShallowPlayButton(props) {
+    const intl = createIntl({
+        locale: 'en',
+        defaultLocale: 'en',
+        messages: messages.en
+    });
+
+    const wrapper = shallow(
+        React.createElement(
+            PlayButton.WrappedComponent,
+            Object.assign(
+                {},
+                defaultPlayButtonProps,
+                {
+                    intl: intl
+                },
+                props
+            )
+        )
+    );
+
+    return wrapper;
+}
+
+function getRunButton(playButtonWrapper) {
+    return playButtonWrapper.find(AriaDisablingButton)
+        .filter('.PlayButton');
+}
+
+describe('The Run button class is changed when the program is running', () => {
```

Comment:
There are many places in this file where the button is called the "run button". Replace with "play button".